rmarkdowns/Week_1.Rmd

```diff
@@ -298,6 +336,18 @@ ggplot(data = world) +
 
 ```
 
+```{r Challenge 5}
+
+
+ggplot(data = world) +
+  geom_sf(fill = "#669438", color = "#32481B", size = 0.25) +
+    coord_sf(crs = "+proj=ortho +lat_0=13 +lon_0=145") +
+  ggtitle("Mariana Islands as the Center of the World", subtitle = "Orthographic Azimuthal") +
+  theme_void()
+
```

Comment:
Very nice! Notice that some of the countries are not showing up (e.g. Russia). I'm not sure exactly why this happens, but will try to figure it out. Nevertheless, you were successful in changing the map center!